docs/en/setup/service-agent/java-agent/How-to-enable-kafka-reporter.md

```diff
@@ -2,6 +2,8 @@
 
 The Kafka reporter plugin support report traces, JVM metrics, Instance Properties, and profiled snapshots to Kafka cluster, which is disabled in default. Move the jar of the plugin, `kafka-reporter-plugin-x.y.z.jar`, from `agent/optional-reporter-plugins` to `agent/plugins` for activating.
 
+If you configure to use `compression.type` such as `lz4`, `zstd`, `snappy`, etc., you also need to move the jar of the plugin, `lz4-java-x.y.z.jar` or `zstd-jni-x.y.z.jar` or `snappy-java.x.y.z.jar`, from `agent/optional-reporter-plugins` to `agent/plugins`.
+
```

Comment:
I think you mentioned, even there is no compression.type set, some jars are still needed.

Reply:
Should we reword L3 too? Or, we simply ask users to move all files to the plugins folder?

Reply:
I think you mentioned `lz4`?

Reply:
I tested was found `lz4` jar is not necessary, probably because the `wrapForOutput`、`wrapForInput` function is not called.

https://github.com/apache/kafka/blob/2.4.1/clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java#L388
https://github.com/apache/kafka/blob/2.4.1/clients/src/main/java/org/apache/kafka/common/record/CompressionType.java#L99

Reply:
OK, then it makes sense. Java only loads class when first time call.